src/components/player/index.jsx

```
@@ -0,0 +1,180 @@
+import { IoHeart, IoChatbubbleEllipses, IoSend } from "react-icons/io5";
+import Modal from "../modal/Modal";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  addCommentOnVideo,
+  getVideoAllComments,
+  getVideoAllLikes,
+  likeUnlikeVideo,
+} from "../../app/features/videomania";
+import AppInput from "../form/AppInput";
+
+const Player = ({ video, isOpen, onClose }) => {
```

ESLint

[error] 'video' is missing in props validation

(react/prop-types)